src/libs/shim-proto/proto/github.com/containerd/containerd/api/types/mount.proto

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
It would be helpful if all these files were under a `vendor/` directory to make it clearer that they are "read only" / pristine sources that should not be changed.

Also, I don't see details in the commits of which containerd commit these files are from?